airflow/providers/apache/druid/example_dags/example_druid_dag.py

```diff
@@ -19,22 +19,19 @@
 """
 Example Airflow DAG to submit Apache Druid json index file using `DruidOperator`
 """
+from datetime import datetime
+
 from airflow.models import DAG
 from airflow.providers.apache.druid.operators.druid import DruidOperator
-from airflow.utils.dates import days_ago
 
 with DAG(
     dag_id='example_druid_operator',
     schedule_interval=None,
-    start_date=days_ago(2),
+    start_date=datetime(2021, 1, 1),
     tags=['example'],
 ) as dag:
     # [START howto_operator_druid_submit]
-    submit_job = DruidOperator(
-        task_id='spark_submit_job',
-        json_index_file='json_index.json',
-        druid_ingest_conn_id='druid_ingest_default',
-    )
+    submit_job = DruidOperator(task_id='spark_submit_job', json_index_file='json_index.json')
```

Comment:
Why remove `druid_ingest_conn_id` here, because it's the default?

Reply:
Yes. We've been removing these default connection IDs across the example DAGs. Do you feel differently?